js/views/site.es6.js

```diff
@@ -0,0 +1,45 @@
+const Parent = window.DDG.base.View;
+
+function Site (ops) {
+
+    this.model = ops.model;
+    this.pageView = ops.pageView;
+    this.template = ops.template;
+
+    Parent.call(this, ops);
+
+    console.log("new site view");
+
+
+    // this._cacheElems() caches jQuery selectors, so the following would be
+    // accessible via: `this.$item` from within this view
+    // and is equivalent to $('.js-site-item')
+
+    this._cacheElems('.js-site', [ 'whitelistToggle' ]);
```

Comment:
please use `kebab-case` vs `camelCase` for class names. So `whitelist-toggle` vs what's there.